2d_classification/monai_101.ipynb

```diff
@@ -16,19 +16,31 @@
     "See the License for the specific language governing permissions and  \n",
     "limitations under the License.\n",
     "\n",
-    "# MONAI 101 tutorial\n",
+    "# MONAI 101 Tutorial\n",
     "\n",
-    "In this tutorial, we will introduce how simple it can be to run an end-to-end classification pipeline with MONAI.\n",
+    "Welcome to MONAI 101! This tutorial introduces beginners to the basics of building an end-to-end medical image classification pipeline with MONAI.\n",
```

Comment:
```suggestion
    "This tutorial introduces the basics of building an end-to-end medical image classification pipeline with MONAI.\n",
```
As a personal taste thing I'd remove unnecessary components like this, welcoming statements don't really match the style of other notebooks I'd say anyway.